fridgify_backend/management/commands/fill_db.py

```diff
@@ -0,0 +1,144 @@
+                name=self.fake.last_name(),
+                surname=self.fake.first_name(),
+                email=self.fake.email(),
+                password=pw,
+                birth_date=self.fake.date_of_birth().strftime("%Y-%m-%d")
+            )
+        models.Users.objects.create(
+            username="fridgify_test",
+            name="Fridgify",
+            surname="Test",
+            email="fridgify@fridgify.com",
+            password=pw,
+            birth_date=self.fake.date_of_birth().strftime("%Y-%m-%d")
+        )
+        self.users = models.Users.objects.all()
+
+    def create_fridge(self, count):
+        for i in range(count):
+            fridge = models.Fridges(
+                name=self.fake.word(),
+                description=self.fake.text()
+            )
+            fridge.save()
+        self.fridges = models.Fridges.objects.all()
+
+    def connect_uf(self, user_count, upf=1):
+        ufs = []
+        for fridge in self.fridges:
+            exclude = []
+            ufs.append(models.UserFridge(
+                user=models.Users.objects.get(username="fridgify_test"),
+                fridge=fridge,
+                role=0
+            ))
+            for i in range(upf):
+                rand = random.randint(0, user_count-1)
+                while rand in exclude:
+                    rand = random.randint(0, user_count-1)
+                
+                ufs.append(models.UserFridge(
+                    user=self.users[rand],
+                    fridge=fridge,
+                    role=random.randint(1, 2)
+                ))
+                exclude.append(rand)
+        models.UserFridge.objects.bulk_create(ufs)
+
+    def create_store(self):
+        models.Stores.objects.bulk_create([
+            models.Stores(name="Rewe"),
+            models.Stores(name="Aldi Süd"),
+            models.Stores(name="Penny"),
+            models.Stores(name="Lidl"),
+            models.Stores(name="Nahkauf"),
+        ])
+        self.stores = models.Stores.objects.all()
+
+    def create_item(self, count):
+        for i in range(count):
+            models.Items.objects.create(
+                barcode=self.fake.ean(),
+                name=self.fake.word(),
+                description=self.fake.sentence(),
+                store=self.stores[random.randint(0, (len(self.stores) - 1) )]
+            )
+        self.items = models.Items.objects.all()
+
+    def fill_fridges(self, min_amount, max_items):
+        fc = []
+        units = ["kg", "g", "l", "ml"]
+        count_cid = {}
+        for fridge in self.fridges:
+            for i in range(random.randint(min_amount, max_items)):
+                dates = [
+                    self.fake.future_date().strftime("%Y-%m-%d"),
+                    self.fake.date_time_between(start_date='-20d')
+                ]
+                amount = random.randint(0, 1000)
+                item = self.items[random.randint(0, (len(self.items)-1))]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/gh/Fridgify/Fridgify_Backend/pullRequest?prid=5673485)